salesforce/client.bal

```diff
@@ -95,42 +113,151 @@ public isolated client class Client {
+    # 
+    # # Deprecated
+    # This function is deprecated as the `salesforce` client is deprecated.
+    # Use the `sObjectPlatformAction` function in `salesforce.rest` client instead.
     @display {label: "Get SObject Platform Action"}
+    @deprecated
     isolated remote function sObjectPlatformAction() 
                                                   returns @display {label: "SObject Basic Information"} 
                                                   SObjectBasicInfo|Error {
-        string path = prepareUrl([API_BASE_PATH, SOBJECTS, PLATFORM_ACTION]);
-        json res = check self->getRecord(path);
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS, PLATFORM_ACTION]);
+        json res = check self.get(path);
         return toSObjectBasicInfo(res);
     }
 
+
+    //Describe Organization
+    # Lists summary details about each REST API version available.
+    #
+    # + return - List of `Version` if successful. Else, the occured Error.
+    # 
+    # # Deprecated
+    # This function is deprecated as the `salesforce` client is deprecated.
+    # Use the `getAvailableApiVersions` function in `salesforce.rest` client instead.
+    @display {label: "Get Available API Versions"}
+    @deprecated
+    isolated remote function getAvailableApiVersions() returns @display {label: "Versions"} Version[]|Error {
+        string path = utils:prepareUrl([BASE_PATH]);
+        json res = check self.get(path);
+        return toVersions(res);
+    }
+
+    # Lists the resources available for the specified API version.
+    #
+    # + apiVersion - API version (v37)
+    # + return - `Resources` as map of strings if successful. Else, the occurred `Error`.
+    # 
+    # # Deprecated
+    # This function is deprecated as the `salesforce` client is deprecated.
+    # Use the `getResourcesByApiVersion` function in `salesforce.rest` client instead.
+    @display {label: "Get Resources by API Version"}
+    @deprecated
+    isolated remote function getResourcesByApiVersion(@display {label: "API Version"} string apiVersion) 
+                                                      returns @display {label: "Resources"} map<string>|Error {
+        string path = utils:prepareUrl([BASE_PATH, apiVersion]);
+        json res = check self.get(path);
+        return toMapOfStrings(res);
+    }
+
+    # Lists the Limits information for your organization.
+    #
+    # + return - `OrganizationLimits` as map of `Limit` if successful. Else, the occurred `Error`.
+    # 
+    # # Deprecated
+    # This function is deprecated as the `salesforce` client is deprecated.
+    # Use the `getOrganizationLimits` function in `salesforce.rest` client instead.
+    @display {label: "Get Organization Limits"}
+    @deprecated
+    isolated remote function getOrganizationLimits() 
+                                                   returns @display {label: "Organization Limits"} 
+                                                   map<Limit>|Error {
```

Comment:
There is some formatting issue here.

Reply:
Fixed in 8bfb122ccf270a0868c40a7ea6ddaef1d3a79a39